src/Bridges/scalarizebridge.jl

```diff
@@ -0,0 +1,93 @@
+`AbstactVectorFunction`-in-`vector_set_type(S)`.
+"""
+mutable struct ScalarizeBridge{T, F, S} <: AbstractBridge
+    scalar_constraints::Vector{CI{F, S}}
+end
+function ScalarizeBridge{T, F, S}(model::MOI.ModelLike,
+                                  f::MOI.AbstractVectorFunction,
+                                  set::VectorLPSet) where {T, F, S}
+    dimension = MOI.output_dimension(f)
+    new_f = scalarize(f)
+    constraints = CI{F,S}[]
+    sizehint!(constraints, dimension)
+    for i in 1:dim
+        push!(constraints, MOI.add_constraint(model, new_f[i], vector_set(T, set)))
+    end
+    return ScalarizeBridge{T, F, S}(constraints)
+end
+
+function MOI.supports_constraint(::Type{ScalarizeBridge{T}},
+                                ::Type{<:MOI.AbstractScalarFunction},
+                                ::Type{<:VectorLPSet}) where T
+    return true
+end
+function added_constraint_types(::Type{ScalarizeBridge{T, F, S}}) where {T, F, S}
+    return [(F, S)]
+end
+function concrete_bridge_type(::Type{<:ScalarizeBridge{T}},
+                              F::Type{<:MOI.AbstractVectorFunction},
+                              S::Type{<:VectorLPSet}) where T
+    return ScalarizeBridge{T, vector_function_type(F), vector_set_type(S)}
+end
+
+# Attributes, Bridge acting as an model
+function MOI.get(bridge::ScalarizeBridge{T, F, S},
+                 ::MOI.NumberOfConstraints{F, S}) where {T, F, S}
+    return length(bridge.scalar_constraints)
+end
+function MOI.get(bridge::ScalarizeBridge{T, F, S},
+                 ::MOI.ListOfConstraintIndices{F, S}) where {T, F, S}
+    return bridge.scalar_constraints
+end
+
+# References
+function MOI.delete(model::MOI.ModelLike, bridge::ScalarizeBridge)
+    MOI.delete.(model, bridge.scalar_constraints)
+end
+
+# Attributes, Bridge acting as a constraint
+
+function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintPrimal,
+                 bridge::ScalarizeBridge)
+   MOI.get.(model, attr, bridge.scalar_constraints)
+end
+function MOI.get(model::MOI.ModelLike, attr::MOI.ConstraintDual,
+                 bridge::ScalarizeBridge)
+    MOI.get.(model, attr, bridge.scalar_constraints)
+end
+function MOI.modify(model::MOI.ModelLike, bridge::ScalarizeBridge,
+    change::MOI.VectorConstantChange{T}) where T
+    MOI.modify.(model, bridge.scalar_constraints,
+        MOI.ScalarConstantChange{T}.(change.new_constant))
+end
+function MOI.modify(model::MOI.ModelLike, bridge::ScalarizeBridge,
+                    change::MOI.MultirowChange{T}) where T
+    for (index, value) in change.new_coefficients
+        MOI.modify(model, bridge.scalar_constraints[index],
+               MOI.ScalarCoefficientChange{T}(change.variable, value))
+    end
+    nothing
+end
+function MOI.set(model::MOI.ModelLike, ::MOI.ConstraintFunction,
+    bridge::ScalarizeBridge{T}, func) where T
+    new_func = scalarize(func)
+    MOI.set.(model, MOI.ConstraintFunction(), bridge.scalar_constraints, new_func)
```

Comment:
You should chech if the constant changed and modify the set

Reply:
I made the change regardless for now

Reply:
now its correct